src/ssl.c

```diff
@@ -18838,13 +19028,13 @@ WOLF_STACK_OF(WOLFSSL_X509)* wolfSSL_set_peer_cert_chain(WOLFSSL* ssl)
         if (ret == 0 && i == ssl->session.chain.count-1) {
             /* On the last element in the chain try to add the CA chain
              * first if we have one for this cert */
-            if (pushCAx509Chain(ssl->ctx->cm, x509, sk)
+            if (PushCAx509Chain(SSL_CM(ssl), x509, sk)
                     == WOLFSSL_FATAL_ERROR) {
                 ret = WOLFSSL_FATAL_ERROR;
             }
         }
 #else
-        (void)pushCAx509Chain;
+        (void)PushCAx509Chain;
```

Comment:
Please fix, so `PushCAx509Chain` is not compiled in when `WOLFSSL_QT`. I would prefer the function not be compiled vs using a `(void)` hack to avoid the unused function. Also not sure why QT would do this?

Reply:
This is changed in https://github.com/wolfSSL/wolfssl/pull/4059. I'm not sure why this is compiled out for QT.